BeeTracker/app/src/main/res/values/strings.xml

```diff
@@ -52,12 +52,25 @@
     <string name="error_message_submit">Something went wrong with submitting your image.</string>
     <string name="preference_file_key">com.blueprint.foe.beetracker.PREFERENCE_FILE_KEY</string>
     <string name="preference_login_token">LOGIN_TOKEN</string>
-    <string name="preference_logged_in_as_guest">GUEST_LOGIN</string>
+    <string name="preference_login_token_type">LOGIN_TOKEN_TYPE</string>
+    <string name="preference_login_client">LOGIN_CLIENT</string>
+    <string name="preference_login_expiry">LOGIN_EXPIRY</string>
+    <string name="preference_login_uid">LOGIN_UID</string>
     <string name="preference_first_time_identifying">FIRST_TIME_IDENTIFYING</string>
+    <string name="signup_email">SIGNUP_EMAIL</string>
+    <string name="signup_password">SIGNUP_PASSWORD</string>
+    <string name="email_label">EMAIL</string>
+    <string name="email_placeholder">Enter email</string>
+    <string name="password_label">PASSWORD</string>
+    <string name="password_placeholder">Enter password</string>
+    <string name="forgot_password">Forgot Password?</string>
+    <string name="login">Log In</string>
+    <string name="signup_with_email">Sign Up With Email</string>
+    <string name="or">Or</string>
     <string name="continue_as_guest">Login as guest</string>
-    <string name="login_with_facebook">Login with Facebook</string>
-    <string name="great_canadian">The Great Canadian Bumble Bee Count</string>
-    <string name="foe">Friends of the Earth</string>
+    <string name="signin_with_facebook">Sign in with Facebook</string>
+    <string name="great_bumblebee_count">The Great Bumblebee Count</string>
```

Comment:
I was under the impression we were supposed to keep Canadian in the name. I see it's in the designs though so I'll take it up with @johnsington